buda/entities/signup.proto

```diff
@@ -11,6 +11,8 @@ message Signup {
     Uuid user_id = 2;
     Uuid visitor_id = 3;
     Uuid legacy_visitor_id = 100; //buffer-web visitor_id, will be deprecated
-
+    Uuid client_id = 5;
     google.protobuf.Timestamp created_at = 4;
+    string signup_option = 6;
```

Comment:
Maybe this could be an enum too, since we have a small number of possible values that we can accept (email, facebook, google, linkedin, twitter, org_team_invite, not sure if there are any others?)

Reply:
Ah, yup, good call! I agree that would work well here 👍

Reply:
I went ahead and made that change

Reply:
I think the field is duplicated here? I think this should be something more like this?

```
message Signup {
 enum SignupOption {
      EMAIL = 0;
      FACEBOOK = 1;
      TWITTER = 2;
      LINKEDIN = 3;
      GOOGLE = 4;
      ORG_TEAM_INVITE = 5;
    }
    ...
    SignupOption signup_option = 6;
    bool with_trial = 7;
}
```

Reply:
Ah, I see, thanks for the feedback!! 👍

Reply:
I just committed a change, would love to hear what you think :)